components/LoginForm.vue

```diff
@@ -1,208 +1,179 @@
 <template>
-  <br-q-title-card
-    v-if="!awaitingAuthorization"
-    :title="title"
-    class="full-width">
-    <template #body>
-      <div class="column q-pa-md">
-        <div v-if="showDeviceAlreadyRegistered">
-          <p class="text-left q-mb-none">
-            We detected that your device was already registered. Either you
-            registered your device in another browser tab or you may have been
-            on a website that was pretending to be <strong>{{domain}}</strong>.
-          </p>
-          <p class="text-left q-mt-md">
-            Make sure to check your browser's URL bar on any other tabs that
-            may have directed you here to ensure you are on the correct website
-            and then try to
+  <q-card
+    flat
+    bordered>
+    <q-card-section class="q-mt-lg q-pb-sm text-center">
+      <img
+        v-if="branding.logo"
+        :src="branding.logo"
+        :style="`height: ${branding.logoSize.desktop}; filter:invert(1)`">
+    </q-card-section>
+    <q-card-section class="column text-center text-body1 q-px-xl q-mb-lg">
+      <div v-if="showEmail">
+        <div v-if="showSendEmail">
+          Enter email for a code to log in.
```

Comment:
Does this read strange to anyone else?  Alternatives I can think of are many more words.  Maybe it's fine.

"Enter email you registered with to be sent a log in code." is maybe too much.

Reply:
@davidlehn I'm aiming for as concise verbally and visually as possible while still maintaining clarity. When that's achieved user workflows become much easier and more pleasant to execute. I also looked at other apps like PayPal, CashApp, etc that have similar workflows to get an idea of their log in vernacular.

Reply:
Maybe this shorter revision?
>  Enter your email to receive a login code.

Reply:
@BigBlueHat Ideally it'd be 38 characters or less so we can keep it a on-liner and keep the overall card small enough to look solid on mobile.

Obviously, this could be adjusted and so can the padding of the card, but it does begin to look oversized if we keep a reasonable padding with long text.

I'm also planing on skipping the card container for mobile which will also help. Just wanted to highlight to motive here. If we can keep it to a short one-liner the UX becomes much leaner.

> With text suggested above
>
> <img width="400px" alt="log in" src="https://github.com/digitalbazaar/bedrock-vue-wallet/assets/56396286/5eaf15a9-0a49-412c-9110-1b349020f08c" />

Reply:
@BigBlueHat Ideally it'd be 38 characters or less so we can keep it a one-liner and keep the overall card small enough to look solid on mobile.

Obviously, this could be adjusted and so can the padding of the card, but it does begin to look oversized if we keep a reasonable padding with long text.

I'm also planing on skipping the card container for mobile which will also help. Just wanted to highlight to motive here. If we can keep it to a short one-liner the UX becomes much leaner.

Reply:
`'Enter your email to get a login code.'.length` => 37

Reply:
@dlongley for the win!